### DISCUSSION TITLE
milvus有可能重复消费么

### DISCUSSION THREAD

**Discussion by 131452099900** [milvus-io/milvus#31614] [Q&A and General discussion]
目前，在检索的时候我发现有一种突发情况，就是milvus重复插入了一条数据，不知道是milvus里面的插入任务重复消费了还是我这边业务上的问题，因为我这边是直接获取mysql的自增Id，理论是不会出现重复的Id的。

**Comment by yhmo** [comment]
虽然mysql那边拿到的id是自增，但有没有可能客户端逻辑在调用insert的时候同一条数据调了两次

**Reply by 131452099900** [reply]
好像从代码逻辑上并没有出现这种情况，而且这是偶发情况，概率很低只出现了一次，你们之前有没有碰到过这样的重复消费问题
